CMakeLists.txt

```diff
@@ -14,11 +13,11 @@ SET( ${PROJECT_NAME}_VERSION_MAJOR 0 )
 SET( ${PROJECT_NAME}_VERSION_MINOR 4 )
 SET( ${PROJECT_NAME}_VERSION_PATCH 2 )
 
+SET( ${PROJECT_NAME}_VERSION  "${${PROJECT_NAME}_VERSION_MAJOR}.${${PROJECT_NAME}_VERSION_MINOR}.${${PROJECT_NAME}_VERSION_PATCH}" )
 
-
-# project version
-
-cmake_policy(SET CMP0008 NEW)  
+#--- Define basic build settings -----------------------------------------------
+# Provides install directory variables as defined for GNU software
+include(GNUInstallDirs)
```

Comment:
This makes the libraries now go to lib64, which breaks the check in ilcsoft-install
(And the assignment of the library to MARLIN_DLL)

Reply:
That is true. I have made a PR to iLCInstall to accomodate for the `lib[64]` changes. `MARLIN_DLL` should not really matter for Physsim, because it does not build processors. Those need to be explicitly built from the examples directory.